gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/__.java

```diff
@@ -980,7 +981,18 @@ protected __() {
      * @see GraphTraversal#property(VertexProperty.Cardinality, Object, Object, Object...)
      */
     public static <A> GraphTraversal<A, A> property(final VertexProperty.Cardinality cardinality, final Object key, final Object value, final Object... keyValues) {
-        return __.<A>start().property(cardinality, key, value, keyValues);
+        return __.<A>
+        
+        start().property(cardinality, key, value, keyValues);
+    }
+
+    /**
+     * @see GraphTraversal#property(Object)
+     */
+    public static <A> GraphTraversal<A, A> property(final Object value) {
+        return __.<A>
+
+                start().property(value);
```

Comment:
is this extra spacing here intended? could you please fold both this one and the previous back to one line?

Reply:
No that was an auto-formatting change.